systemvm/patches/debian/config/opt/cloud/bin/set_redundant.py

```diff
@@ -1,47 +0,0 @@
-#!/usr/bin/python
```

Comment:
This is called from `test/systemvm/test_update_config.py`. Not sure what it does and if it works, but at least it is referenced there.